<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LiverVolumetryWidget</class>
 <widget class="qMRMLWidget" name="LiverVolumetryWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>263</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="qMRMLCollapsibleButton" name="ResectionVolumetryGroupWidget">
     <property name="text">
      <string>Resection Volumetry</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="3">
       <widget class="qMRMLSegmentationShow3DButton" name="SegmentationShow3DButton"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>ROI Markers List:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="ResectionLabel">
        <property name="text">
         <string>Resection:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="label_1">
        <property name="text">
         <string>Reference Volume:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="qMRMLSegmentSelectorWidget" name="TargetSegmentationSelectorWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The volume of these segments will account as total volume for the volumetry calculation. If no segments are selected, all segments will be considered for the total volume calculation&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="noneEnabled">
         <bool>false</bool>
        </property>
        <property name="segmentationNodeSelectorVisible">
         <bool>false</bool>
        </property>
        <property name="multiSelection">
         <bool>true</bool>
        </property>
        <property name="selectNodeUponCreation">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="2" colspan="2">
       <widget class="qMRMLCheckableNodeComboBox" name="ResectionTargetNodeComboBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If selected ,the resection(s) will be used as boundary for volume calculation&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLLiverResectionNode</string>
         </stringlist>
        </property>
        <property name="noneEnabled">
         <bool>true</bool>
        </property>
        <property name="addEnabled">
         <bool>false</bool>
        </property>
        <property name="removeEnabled">
         <bool>false</bool>
        </property>
        <property name="editEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="qMRMLNodeComboBox" name="ROIMarkersListSelector">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;List of fiducial seeds that used for volume calculation&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLMarkupsFiducialNode</string>
         </stringlist>
        </property>
        <property name="showChildNodeTypes">
         <bool>false</bool>
        </property>
        <property name="baseName">
         <string>ROIMarkersList</string>
        </property>
        <property name="noneEnabled">
         <bool>true</bool>
        </property>
        <property name="editEnabled">
         <bool>true</bool>
        </property>
        <property name="renameEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="qMRMLNodeComboBox" name="VolumeTableSelectorWidget">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Sets the output table node where the volume calculation will be added&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLTableNode</string>
         </stringlist>
        </property>
        <property name="showChildNodeTypes">
         <bool>false</bool>
        </property>
        <property name="baseName">
         <string>LiverResectionVolumetry</string>
        </property>
        <property name="noneEnabled">
         <bool>true</bool>
        </property>
        <property name="editEnabled">
         <bool>true</bool>
        </property>
        <property name="renameEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Segmentation:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="qMRMLSegmentSelectorWidget" name="InputSegmentSelectorWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Segment to account for volume calculation&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="segmentationNodeSelectorVisible">
         <bool>false</bool>
        </property>
        <property name="multiSelection">
         <bool>true</bool>
        </property>
        <property name="selectNodeUponCreation">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_6">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Volumetry Output Table:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="qSlicerMarkupsPlaceWidget" name="ROIMarkersListPlaceWidget">
        <property name="maximumSize">
         <size>
          <width>15000</width>
          <height>40</height>
         </size>
        </property>
        <property name="placeMultipleMarkups">
         <enum>qSlicerMarkupsPlaceWidget::ForcePlaceMultipleMarkups</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="qMRMLNodeComboBox" name="InputSegmentationSelector">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Segmentation for volume calculation&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLSegmentationNode</string>
         </stringlist>
        </property>
        <property name="showChildNodeTypes">
         <bool>false</bool>
        </property>
        <property name="noneEnabled">
         <bool>true</bool>
        </property>
        <property name="addEnabled">
         <bool>false</bool>
        </property>
        <property name="removeEnabled">
         <bool>false</bool>
        </property>
        <property name="editEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Total Volume:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="qMRMLNodeComboBox" name="ReferenceVolumeSelector">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Sets the reference volume which the segmentation was based on&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLScalarVolumeNode</string>
         </stringlist>
        </property>
        <property name="addEnabled">
         <bool>false</bool>
        </property>
        <property name="removeEnabled">
         <bool>false</bool>
        </property>
        <property name="editEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="4">
       <widget class="QPushButton" name="ComputeVolumePushButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Calculate Volume</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLCheckableNodeComboBox</class>
   <extends>qMRMLNodeComboBox</extends>
   <header>qMRMLCheckableNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLCollapsibleButton</class>
   <extends>ctkCollapsibleButton</extends>
   <header>qMRMLCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qSlicerMarkupsPlaceWidget</class>
   <extends>qSlicerWidget</extends>
   <header>qSlicerMarkupsPlaceWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLSegmentSelectorWidget</class>
   <extends>qMRMLWidget</extends>
   <header>qMRMLSegmentSelectorWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLSegmentationShow3DButton</class>
   <extends>ctkMenuButton</extends>
   <header>qMRMLSegmentationShow3DButton.h</header>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkMenuButton</class>
   <extends>QPushButton</extends>
   <header>ctkMenuButton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>LiverVolumetryWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>VolumeTableSelectorWidget</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LiverVolumetryWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>ResectionTargetNodeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>365</y>
    </hint>
    <hint type="destinationlabel">
     <x>336</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>InputSegmentationSelector</sender>
   <signal>currentNodeChanged(vtkMRMLNode*)</signal>
   <receiver>InputSegmentSelectorWidget</receiver>
   <slot>setCurrentNode(vtkMRMLNode*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>422</x>
     <y>576</y>
    </hint>
    <hint type="destinationlabel">
     <x>422</x>
     <y>631</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ROIMarkersListSelector</sender>
   <signal>currentNodeChanged(vtkMRMLNode*)</signal>
   <receiver>ROIMarkersListPlaceWidget</receiver>
   <slot>setCurrentNode(vtkMRMLNode*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>422</x>
     <y>686</y>
    </hint>
    <hint type="destinationlabel">
     <x>762</x>
     <y>686</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LiverVolumetryWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>ROIMarkersListSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>428</x>
     <y>490</y>
    </hint>
    <hint type="destinationlabel">
     <x>422</x>
     <y>686</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LiverVolumetryWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>InputSegmentSelectorWidget</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>428</x>
     <y>490</y>
    </hint>
    <hint type="destinationlabel">
     <x>422</x>
     <y>631</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LiverVolumetryWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>ROIMarkersListPlaceWidget</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>428</x>
     <y>490</y>
    </hint>
    <hint type="destinationlabel">
     <x>762</x>
     <y>686</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LiverVolumetryWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>InputSegmentationSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>428</x>
     <y>490</y>
    </hint>
    <hint type="destinationlabel">
     <x>422</x>
     <y>576</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LiverVolumetryWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>ReferenceVolumeSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>395</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>InputSegmentationSelector</sender>
   <signal>currentNodeChanged(vtkMRMLNode*)</signal>
   <receiver>TargetSegmentationSelectorWidget</receiver>
   <slot>setCurrentNode(vtkMRMLNode*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>422</x>
     <y>576</y>
    </hint>
    <hint type="destinationlabel">
     <x>422</x>
     <y>631</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LiverVolumetryWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>TargetSegmentationSelectorWidget</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>395</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
